.dockerignore

```diff
@@ -1,3 +1,6 @@
 *.pyc
 .idea
 .git
+.tox
+docs/
+.cache
```

Comment:
This file is for things to skip and not put inside the image when `docker build` is executing `COPY . /lewis`. Looking at this file and `.gitignore`, they are quite similar but have subtle differences.

It makes sense for this file to additionally have `.git` because the image doesn't need the whole repo. But I can't think of a reason for the other differences:
- `.coverage` in gitignore but not here
- `docs/_build` vs `docs/`
- `src/lewis.egg-info` in gitignore but not here

Would it make sense to get rid of these differences to make the ignore files more uniform?